ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/model.hpp

```diff
@@ -18,28 +18,35 @@
 
 #include <frontend_manager/frontend_manager.hpp>
 #include <fstream>
+#include <string>
 
 #include "place.hpp"
 
 namespace ngraph {
 namespace frontend {
 
-class FrontEndPDPD;
-
 class NGRAPH_API InputModelPDPD : public InputModel
 {
-    // TODO: replace it by already deserialized proto hidden under some Impl class
-    // TODO: avoid using explicit format-dependent data stuctures here, hide it under some Impl class
-   
     friend class FrontEndPDPD;
+    class InputModelPDPDImpl;
+    std::shared_ptr<InputModelPDPDImpl> _impl;
 
-public:
-    std::string path;
-    std::string model_file;
-    std::ifstream weights_stream;
-    bool weights_composed = false;
+    //template<typename T>
+    std::vector<float> getWeight(const std::string& name, int64_t tensor_length);
```

Comment:
can be 'const'?

Reply:
No, it modifies file stream stored inside _impl in case of concatenated weights